source/dub/packagesuppliers/packagesupplierlist.d

```diff
@@ -0,0 +1,131 @@
+module dub.packagesuppliers.packagesupplierlist;
```

Comment:
Module DDOC please

Reply:
Also it's already in the `packagesuppliers` package, which is a mouthful, so maybe just `list` ?
I'm not so strongly opinionated on this, just find the repetition pointless.